.description.md

```diff
@@ -0,0 +1,4 @@
+The Notification Service is a microservice intended to provide general notification capabilities. At this time, notifications are only generated via Kafka events, and they are only issued via email.
+However, the architecture of the service would allow for the addition of other event sourcing options, like API submission, as well as new notification channels, such as SMS, with relatively little work.
```

Comment:
```suggestion
However, the architecture of the service would allow for the addition of other submission opions, such as REST APIs, as well as new notification channels, such as SMS, with relatively little work.
```

Reply:
Applied